### ISSUE TITLE
Machine readable format

### ISSUE THREAD

**Issue by aantonop** [nvk/wallets-recovery#115]
It would be great if this site could also host a machine readable format (probably JSON) that allows recovery software to read information about wallets, paths and script types and uses the data to search the tree produced by a seed or xpub. 

I'm working on such a recovery tool and would love to have a well managed location for that data that can be updated collaboratively. 

For example:
```
{
wallets_recovery: [
 "ColdCard Mk1": [{ "path" : "m/44'/0'/0'", "script": "p2pkh", "passphrase": "optional", "psbt": true},
                             { "path" : "m/84'/0'/0'", ... etc
]
}
```

**Comment by nvk** [comment]
Yes, that would live in a separate document for each wallet. Want to take a stab at it?

**Comment by seljuck** [comment]
I would be willing to tackle this. I will put out the caveat that I've never done anything on GitHub before. I have worked with JSON files however.

**Comment by seljuck** [comment]
I have created an initial version of walletsrecovery.json. I would like to create a pull request for it. Just checking on the accepted procedure for initiating this.

Thanks

**Comment by Giszmo** [comment]
[The Pull Request](https://github.com/nvk/wallets-recovery/pull/138) is where the discussion is going on now.
